lib/login/login.dart

```diff
@@ -122,6 +131,14 @@ class _LoginPageState extends State<LoginPage> {
           ),
         ),
       );
+      if (repoNumber != 200) {
```

Comment:
Maybe check if repoNumber == 500 so in the case where error 404 will not force the user to create a new account when the network is down.

Reply:
@tonynguyen98 Checked for response code 404, where it will throw an exception. Line 134 was reworked into checking for response code 500.

Reply:
There are a couple of response codes you need to check out for. For example, 403 should def be another to check for: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes